src/jit/gentree.cpp

```diff
@@ -9943,6 +9943,28 @@ void Compiler::gtDispRegVal(GenTree* tree)
         }
     }
 
+#if FEATURE_MULTIREG_RET
+    if (tree->IsCopyOrReload())
+    {
+        for (int i = 0; i < MAX_RET_REG_COUNT - 1; i++)
+        {
+            regNumber reg = (regNumber)tree->AsCopyOrReload()->gtOtherRegs[i];
+            if (reg == REG_NA)
+            {
+                break;
+            }
+            printf(",%s", compRegVarName(reg));
+        }
+    }
+#endif
```

Comment:
> Can you explain the scenario where this code kicks in but the code above under tree->IsCopyOrReloadOfMultiRegCall() does not?

It looks like that `IsCopyOrReloadOfMultiRegCall()` is for the CopyOrReload node and its op1 is **multiple return registers**. 

> Is that scenario ARM specific?

This is ARMEL specific scenario. On hardfp, it will use d0, d1, ...(which are treated as one register) so it would not have the same problem with armel.